###

Supplemental Electronic Data files for:

Purifying selection and adaptive evolution proximate to the zoonosis of SARS-CoV-1 and SARS-CoV-2 

Jeffrey P. Townsend[^1],[^2],[^3],\*, Stephen Gaughran[^3],[^4], Hayley B. Hassler[^1],[^5], J. Nicholas Fisk[^2], Mofeed Nagib[^1], Yinfei Wu[^1], Yaning Wang[^1],[^6], Zheng Wang[^1], Lauren A. Meyers[^7], Alison P. Galvani[^8], Alex Dornburg[^9],\*

```
[^1]: Department of Biostatistics, Yale School of Public Health, New Haven, Connecticut 06510, USA
[^2]: Program in Computational Biology and Bioinformatics, Yale University, New Haven, Connecticut 06511, USA
[^3]: Department of Ecology and Evolutionary Biology, Yale University, New Haven, Connecticut 06525, USA
[^4]: Department of Ecology and Evolutionary Biology, Princeton University, Princeton, New Jersey 08544, USA
[^5]: Interdisciplinary Graduate Program in Quantitative Biosciences, Georgia Institute of Technology, Atlanta, GA 30332, USA
[^6]: Institute of Microbiology, Chinese Academy of Sciences, Beijing 100101, China
[^7]: Department of Integrative Biology, University of Texas at Austin, Austin, TX 78712, USA
[^8]: Center for Infectious Disease Modeling and Analysis, Department of Epidemiology of Microbial Disease, Yale School of Public Health, New Haven, Connecticut 06525, USA
[^9]: Department of Bioinformatics and Genomics, University of North Carolina, Charlotte, NC 28223, USA


\* Corresponding authors
  Jeffrey P. Townsend
  135 College St, New Haven, CT 06510-2483. jeffrey.townsend@yale.edu, (203) 737-7042
  Alex Dornburg
  9331 Robert D. Snyder Rd, Room 357, Charlotte, NC 28223. adornbur@uncc.edu, (704) 687-8437
```
Electronic Data Files prepared by:
Hayley B. Hassler
hayley.hassler@yale.edu


###

This directory contains MASS-PRF input and output files, PAML input and output files, newick tree files, phylogenetic informativeness input files and scripts, and link to mass-prf-protein-coloring scripts.

~/MASSPRF_input/ contains fast alignment files for all SARS-CoV-1 and SARS-CoV-2 loci used for MASS-PRF analysis.

~/PAML_input/ contains fast alignment files and newick files for all SARS-CoV-1 and SARS-CoV-2 loci used for PAML analysis.

~/MASSPRF_results/ contains output files generated by MASS-PRF analysis.

~/PAML_results/ contains output files generated by PAML analysis.

~/Phylogenetic_informativeness/ contains input site rates, guide tree, and R scripts for phylogenetic informativeness analyses.

~/Phylogenies/ contains newick files for molecular RAxML, molecular IQ-TREE 2, time-dated IQ-TREE 2, and time-dated RTDT phylogenies.

Link to [mass-prf-protein-coloring](https://github.com/Townsend-Lab-Yale/massprf-protein-coloring) scripts.


